userspace/libsinsp/utils.cpp

```diff
@@ -1417,3 +1417,28 @@ bool set_socket_blocking(int sock, bool block)
 	}
 	return true;
 }
+
+unsigned int read_num_possible_cpus(void)
+{
+	static const char *fcpu = "/sys/devices/system/cpu/possible";
+	unsigned int start, end, possible_cpus = 0;
+	char buff[128];
+	FILE *fp;
+
+	fp = fopen(fcpu, "r");
+	if (!fp) {
+		printf("Failed to open %s: '%s'!\n", fcpu, strerror(errno));
+		exit(1);
```

Comment:
We probably don't want to exit the process right? :)

Also logging to stdout might be less than ideal.